pkgs/development/python-modules/opytimark/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, coverage
+, fetchFromGitHub
+, numpy
+, pre-commit
+, pylint
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "opytimizer";
+  version = "1.0.8";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "gugarosa";
+    repo = "opytimark";
+    rev = "v${version}";
+    hash = "sha256-T3OFm10gvGrUXAAHOnO0Zv1nWrXPBXSmEWnbJxrWYU0=";
+  };
+
+  nativeBuildInputs = [ coverage pre-commit pylint ];
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  # several tests are failing
+  disabledTests = [
+    "test_year"
+    "test_decorator"
+    "test_loader"
+    "cec_benchmark"
+  ];
+
+  pythonImportsCheck = [
+    "opytimark"
+  ];
+
+  meta = with lib; {
+    description = "Opytimark is a Python library consisting of optimization benchmarking functions";
```

```suggestion
    description = "Library consisting of optimization benchmarking functions";
```

See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes